src/runtime/virtcontainers/hypervisor_amd64.go

```diff
@@ -18,8 +18,10 @@ func availableGuestProtection() (guestProtection, error) {
 	if d, err := os.Stat(tdxSysFirmwareDir); (err == nil && d.IsDir()) || flags[tdxCPUFlag] {
 		return tdxProtection, nil
 	}
-
-	// TODO: Add support for other technologies: SEV
+	// SEV is supported and properly loaded when `/sys/module/kvm_amd/parameters/sev` is set to `1`
+	if _, err := os.Stat(sevKvmParameter); err == nil && !os.IsNotExist(err) {
```

Comment:
It doesn't make sense to check err details after making sure it is nil. Also, the content of `sevKvmParameter` should be checked.

Reply:
@bergwolf  I've pushed a cleaner & improved version of the SEV support check:
```	
// SEV is supported and enabled when the kvm module `sev` parameter is set to `1`
	if _, err := os.Stat(sevKvmParameterPath); err == nil {
		if c, err := ioutil.ReadFile(sevKvmParameterPath); err == nil && len(c) > 0 && c[0] == '1' {
			return sevProtection, nil
		}
	}
```